sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -102,81 +104,68 @@ export interface Response {
 }
 
 /**
- * Represents properties of the namespace.
+ * @interface
+ * An interface that tracks the settings for paged iteration
  */
-export interface NamespacePropertiesResponse extends NamespaceProperties, Response {}
+export interface PageSettings {
+  /**
+   * @member {number} [continuationToken] The token that keeps track of where to continue the iterator
```

Comment:
From the previous comment it sounded like sometimes a user might provide this on their own (e.g. not as a continuation from the service, but because they wanted to skip ahead), maybe a better comment here to remark that it is the number of items to skip?

Reply:
Otherwise I don't know how they would know how to provide one

Reply:
> I'm still a little confused about if we expect users to treat the continuation token/marker as something they can reason about (e.g. they know it's the event count to skip and sometimes set that based on their own decisions) or if it's only something they hand back to us after we provide it to them.
The reason to prefer opaque strings for continuation tokens is in the future you may need to encode more metadata (e.g. a topic name, a namespace, whatever) and if the contract is a number you can't change/expand that.

While talking to Brian before, we agreed on using skip as the continuationToken, but did not discuss the type or the name of the attribute.

> If it really is "We want to model this as the skip count" why call it a "marker" or "continuationToken" at all and not just refer to it as skip in the page settings?

Would it be a deviation from the rest of the SDKs? (To call it anything other than continuationToken)

Reply:
With the above discussions, I feel like we'd be better off if I pick strings and not advertise the continuationToken as the skip param.
- This will get rid of the new PageSettings interface
- Since I won't be advertising continuationToken as "skip" attribute, there is no need to change its name
- If the service behavior changes, continuationToken being a string would help to allow us to make non-breaking changes
- This would be consistent with the other SDKs.